app/models/manageiq/providers/ansible_tower/automation_manager/tower_api.rb

```diff
@@ -1,6 +1,11 @@
 module ManageIQ::Providers::AnsibleTower::AutomationManager::TowerApi
   extend ActiveSupport::Concern
 
+  def self.included(model)
+    model.include SupportsFeatureMixin
+    model.supports :create
+  end
```

Comment:
We have ActiveSupport::Concern, so we should do:

```suggestion
  included do
    include SupportsFeatureMixin
    supports :create
  end
```